Purchases/Public/RCEntitlementInfos.m

```diff
@@ -0,0 +1,91 @@
+//
+// Created by César de la Vega  on 2019-07-24.
+//
+
+#import "RCEntitlementInfos.h"
+#import "RCEntitlementInfo.h"
+
+
+@interface RCEntitlementInfos ()
+@property (readwrite) NSDictionary<NSString *, RCEntitlementInfo *> *all;
+@end
+
+@implementation RCEntitlementInfos
+
+- (instancetype)initWithEntitlements:(NSDictionary<NSString *, NSDictionary *> *)entitlements forPurchases:(NSDictionary<NSString *, id> *)purchases withDateFormatter:(NSDateFormatter *)dateFormatter withRequestDate:(NSDate *)requestDate
```

Comment:
`Entitlements` and `Purchases` are pretty generic things. 

`EntitlementInfosResponse` and `PurchaserInfoResponse` might make it more clear that this is a parsing method.